go.mod

```diff
@@ -45,7 +45,8 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
-	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208 // indirect
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
```

Comment:
unless this PR requires that, I'd rather you'd avoid updating the go.mod/go.sum with benign updates.
( there is minor risk in doing that, and unless it's needed, I'd rather avoid that )